Sources/ArgumentParser/Usage/UsageGenerator.swift

```diff
@@ -343,33 +343,29 @@ extension ErrorMessageGenerator {
     let valueName = arguments(for: key).first?.valueName
     
     // We want to make the "best effort" in producing a custom error message.
-    // We favour `LocalizedError.errorDescription` and fall back to
+    // We favour `LocalizedError.errorDescription` and fallback to
     // CustomStringConvertible. To opt-in return your custom error mesage
     // in the `description` property of CustomStringConvertible.
-    let customErrorMessage: String? = {
-        switch error {
-            case is LocalizedError where (error as? LocalizedError)?.errorDescription != nil:
-                return (error as? LocalizedError)?.errorDescription
-            case let err?:
-                return String(describing: err)
-            default:
-                return nil
-        }
+    let customErrorMessage: String = {
+      switch error {
+      case is LocalizedError where (error as? LocalizedError)?.errorDescription != nil:
+        return ": " + (error as! LocalizedError).errorDescription! // !!! Checked above that this will not be nil
```

Comment:
This could be written as:
```
case let err as LocalizedError where err.errorDescription != nil:
    return ": " + err.errorDescription
```

Reply:
I'm not sure why I have such a blind spot for switch/case patterns!